tests/unit/BaseAmqpTest.php

```diff
@@ -2,87 +2,95 @@
 
 namespace RabbitMqModule;
 
-class BaseAmqpTest extends \PHPUnit\Framework\TestCase
+use PhpAmqpLib\Channel\AMQPChannel;
+use PhpAmqpLib\Connection\AbstractConnection;
+use PhpAmqpLib\Wire\AMQPTable;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use RabbitMqModule\Options\Exchange as ExchangeOptions;
+use RabbitMqModule\Options\Queue as QueueOptions;
+
+class BaseAmqpTest extends TestCase
 {
     public function testConstructor()
     {
-        $connection = $this->getMockBuilder('PhpAmqpLib\\Connection\\AbstractConnection')
+        $connection = $this->getMockBuilder(AbstractConnection::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $channel = $this->getMockBuilder('PhpAmqpLib\\Channel\\AMQPChannel')
+
+        $channel = $this->getMockBuilder(AMQPChannel::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $baseAmqp = $this->getMockBuilder('RabbitMqModule\\BaseAmqp')
+
+        /** @var MockObject|BaseAmqp $baseAmqp */
+        $baseAmqp = $this->getMockBuilder(BaseAmqp::class)
             ->setConstructorArgs([$connection])
             ->setMethods(['__destruct'])
             ->getMock();
 
-        $baseAmqp->method('__destruct');
-
-        $connection->expects(static::once())->method('channel')->willReturn($channel);
+        $connection->expects(static::once())
+            ->method('channel')
+            ->willReturn($channel);
 
-        /** @var \RabbitMqModule\BaseAmqp $baseAmqp */
         static::assertEquals($channel, $baseAmqp->getChannel());
     }
 
     public function testSetChannel()
     {
-        $connection = $this->getMockBuilder('PhpAmqpLib\\Connection\\AbstractConnection')
+        $connection = $this->getMockBuilder(AbstractConnection::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $channel = $this->getMockBuilder('PhpAmqpLib\\Channel\\AMQPChannel')
+
+        $channel = $this->getMockBuilder(AMQPChannel::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $baseAmqp = $this->getMockBuilder('RabbitMqModule\\BaseAmqp')
+
+        /** @var MockObject|BaseAmqp $baseAmqp */
+        $baseAmqp = $this->getMockBuilder(BaseAmqp::class)
             ->setConstructorArgs([$connection])
             ->setMethods(['__destruct'])
             ->getMock();
 
-        $baseAmqp->method('__destruct');
-
-        /* @var \RabbitMqModule\BaseAmqp $baseAmqp */
+        /** @var MockObject|AMQPChannel $channel */
         $baseAmqp->setChannel($channel);
-        static::assertEquals($channel, $baseAmqp->getChannel());
-    }
-
-    protected static function getMethod($name)
-    {
-        $class = new \ReflectionClass('RabbitMqModule\\BaseAmqp');
-        $method = $class->getMethod($name);
-        $method->setAccessible(true);
 
-        return $method;
+        static::assertEquals($channel, $baseAmqp->getChannel());
```

Comment:
It's not a test, and I think we can remove this method because I don't see any usage.

Reply:
Can we use `$this->getMockBuilder(AbstractConnection::class)` syntax in test? As i can see you always use `$this->getMockBuilder('PhpAmqpLib\\Connection\\AbstractConnection')`

Reply:
> It's not a test, and I think we can remove this method because I don't see any usage.

Are you mean `testSetChannel` or `getMethod` ? If last it is already deleted.

Reply:
You're right! My mistake! Thanks.